docs/synthetics/metrics.md

```diff
@@ -122,3 +122,36 @@ module.exports = testPage;
 The charting and alerting process is identical as in the previous example, and once again you will be able to see the custom metrics you've defined in the **Metrics** dropdown.
 
 ![Custom Metrics JSON](../images/synthetics/custom-metrics-json.png)
+
+## How to add availability as metric in Dashboards
+
+Dashboards are your central location where you put everything together. Logs, service metrics, infrastructure inventory and processes. They let you see everything in one place. If you want to see any of your monitor’s availability within a Dashboard and compare with performance metrics that are shipped from services hosting your websites and APIs here is how you should do it.
+
+Availability is shown for each HTTP and Browser monitor within All Monitors page and Overview pages of monitors.
+
+![All Monitors Availability](../images/synthetics/availability-all-monitors-page.png)
+
+When you want to add monitor availability as a metric to any of your [Dashboards](https://sematext.com/docs/dashboards/), you can use [Chart Builder](../dashboards/chart-builder/)’s  transformation feature to do it. 
+
+To do this, navigate to the Dashboards page and click on any of your existing Dashboards or create a new one. Then click to “add component” on the top right corner of the report.
```

Comment:
Then click to => Then click on
on the top right corner => in the top-right corner